code/drasil-printers/lib/Language/Drasil/HTML/Helpers.hs

```diff
@@ -131,10 +130,11 @@ descWrap :: [String] -> Doc -> Doc -> Doc
 descWrap = flip (wrapGen Class "dt")
 
 -- | Helper for wrapping divisions or sections.
+-- | Arguments: Wrapper text, attribute value, body text
```

Comment:
Thank you for adding the note! I'm just going to be pedantic with one thing: "Wrapper text" -> "Wrapper element type/tag (e.g., p, div, a)" is a bit better.